docs/en_US/Installation.md

```diff
@@ -1,8 +1,8 @@
 # Installation of NNI
 
-Currently we only support installation on Linux & Mac.
+Currently we support installation on Linux Mac & Windows.
```

Comment:
"Linux Mac & Windows." can change to "Linux, Mac and Windows." or something else.